src/components/SideBar/index.tsx

```diff
@@ -1,14 +1,59 @@
 import React from 'react';
 
 import './styles.scss';
+import { connect } from 'react-redux';
+import classNames from 'classnames';
+import { AppState } from '../../redux/reducer';
+import { AppAction, SectionName } from '../../redux/actionTypes';
+import { changeSection } from '../../redux/actions';
+import { DispatchProps } from '../dispatchProps';
 
-const SideBar: React.FunctionComponent = () => (
-  <div className="menu-container">
-    <div className="app-name">Helpe</div>
-    <div className="section-text">Bug Reports</div>
-    <div className="section-text selected-section">Customer Service</div>
-    <div className="section-text">Feature Requests</div>
-  </div>
-);
+interface SideBarProps extends DispatchProps {
+  sectionName: string
+}
 
-export default SideBar;
+class SideBar extends React.Component<SideBarProps> {
+  sectionClicked(section: SectionName) {
+    const { dispatch } = this.props;
+    dispatch(changeSection(section));
+  }
+
+  render() {
+    const { sectionName } = this.props;
+    return (
+      <div className="background">
+        <div className="menu-container">
+          <div className="app-name">Helpe</div>
+          <div
+            className={classNames('section-text', { 'selected-section': sectionName === SectionName.BugReports })}
```

Comment:
maybe declare section-text and selected-section as const at the top of the file: `const SECTION_TEXT = 'section-text'`